src/test/java/core/basesyntax/service/impl/FileReaderServiceImplTest.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.service.FileReaderService;
+import org.junit.Test;
+
+public class FileReaderServiceImplTest {
+    private static final FileReaderService fileReaderService = new FileReaderServiceImpl();
+    private static final String VALID_PATH = "src/test/resources/test.csv";
+
+    @Test
+    public void readFromFile_ValidPath_ok() {
+        String actual = fileReaderService.readFromFile(VALID_PATH);
+        String expected = "Hello, my friend! Approve this task, please)"
+                + System.lineSeparator() + "-_-";
+        assertEquals("data from " + VALID_PATH + " should be read", expected, actual);
+    }
+
+    @Test(expected = RuntimeException.class)
```

Comment:
I think it's bad idea to expect general RuntimeException as it can be any exception (includes NPE)
better to create your own custom exceptions